manifest/reader.go

```diff
@@ -282,6 +282,13 @@ func ValidateModules(mods *pbsubstreams.Modules) error {
 
 		for idx, in := range mod.Inputs {
 			switch i := in.Input.(type) {
+			case *pbsubstreams.Module_Input_Params_:
+				if idx != 0 {
+					return fmt.Errorf("module %q: input %d: params must be first input", mod.Name, idx)
+				}
+				if i.Params.Value == "" {
```

Comment:
It's perfectly fine that some modules have the empty string for parameters. Just skip this check.